logstash/tests/docker/logstash5/logstash.yml

```diff
@@ -0,0 +1,200 @@
+# Settings file in YAML
```

Comment:
I don't think we need all these examples in the config file, let's keep a few lines only

Reply:
That makes sense, @coignetp. Other than that, it looks good.

Reply:
Removed all comment lines